File: nup/src/main/java/StatsRow.java
// Copyright 2021 Daniel Erat <dan@erat.org>
// All rights reserved.

package org.erat.nup;

// Holds song count for a specific key.
class StatsRow {
    public final StatsKey key;
    public final int count;

    public StatsRow(StatsKey key, int count) {
        this.key = key;
        this.count = count;
    }

    public StatsRow(String artist, String album, String albumId, int count) {
        this(new StatsKey(artist, album, albumId), count);
    }
}

File: nup/src/main/java/StatsRow.kt
// Copyright 2021 Daniel Erat <dan@erat.org>
// All rights reserved.
package org.erat.nup

// Holds song count for a specific key.
class StatsRow(
    val key: StatsKey,
    var count: Int
) {
    constructor(artist: String, album: String, albumId: String, count: Int) :
        this(StatsKey(artist, album, albumId), count) {}
}
